MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp

```diff
@@ -1474,54 +1504,141 @@ EnggDiffractionViewQtGUI::splitFittingDirectory(std::string &selectedfPath) {
   return splitBaseName;
 }
 
+void MantidQt::CustomInterfaces::EnggDiffractionViewQtGUI::enableMultiRun(
+    std::string firstRun, std::string lastRun) {
+
+  bool firstDig = isDigit(firstRun);
+  bool lastDig = isDigit(lastRun);
+
+  std::vector<std::string> RunNumberVec;
+  if (firstDig && lastDig) {
+    int firstNum = std::stoi(firstRun);
```

Comment:
The `stoi` and similar functions will raise an exception if they cannot do the conversion. You could use that and probably wouldn't need the `isDigit()` checks.